bin/setupdb.sh

```diff
@@ -54,8 +55,8 @@ echo
 echo "Database configuration"
 echo
 
-[[ -z $DBNAME ]] && read -p "New OPUS Database Name [opusdb]: "           DBNAME
-[[ -z $DB_ADMIN ]] && read -p "New OPUS Database Admin Name [opus4admin]: " DB_ADMIN
+[[ -z $DBNAME ]] && read -p "New OPUS Database Name [opusdbFW]: "           DBNAME
```

Comment:
I suggest not using capital letter. There is no technical reason. It is just one more thing to remember. Lets keep it all small here.

Reply:
Done.